src/wallet/ismine.h

```diff
@@ -14,7 +14,26 @@
 class CWallet;
 class CScript;
 
-/** IsMine() return codes */
+/**
+ * IsMine() return codes, which depend on ScriptPubKeyMan implementation.
+ * Not every ScriptPubKeyMan covers all types, please refer to
+ * doc/release-notes.md#ismine-semantics for better understanding.
+ * 
```

Comment:
Looks like the trailing whitespace on this line, line 30, and line 35 are tripping the whitespace linter.